### ISSUE TITLE
当我使用透明Activity时，退出透明Activity时Apng会卡主线程，造成OnDestory延迟调用

### ISSUE THREAD

**Issue by fuxuanyang11** [penfeizhou/APNG4Android#181]
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/penfeizhou/APNG4Android/issues) and found none

### Issue Info 
如图
![K0D~% )ZVO4GA{KYVZXT5X](https://user-images.githubusercontent.com/21265701/232994377-53551913-87b5-4c4a-8afa-ebfb9a86c42f.png)

| Info                         | Value                                                 |      |
| ---------------------------- | ----------------------------------------------------- | ---- |
| Device Info                  | 任何手机                              |      |
| System Version               | e.g.  6.0                                             |      |
| APNG4Android Library Version | 2.24.0                                           |      |
| Repro rate                   | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo project  | e.g. does it happen with our demo project?            |      |
| Demo project link            | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by jingpeng** [comment]
Android App开发中onDestroy本就不会立刻调用，系统自行决定onDestroy调用时机
